### PR TITLE
boards/nucleo-l452re: add pinout to doc page

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
#### Contribution description

This PR adds missing pinout diagram from board manual mb1136 for nucleo-l452re board.

### Testing procedure

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-l452re.html
```

### Issues/PRs references

None

### CODE REVIEW

File: boards/nucleo-l452re/doc.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@
 The Nucleo-L452RE is a board from ST's Nucleo family supporting ARM Cortex-M4
 STM32L452RE microcontroller with 160KiB of RAM and 512KiB of Flash.
 
+## Pinout
+
+@image html pinouts/nucleo-l452re.svg "Pinout for the nucleo-f452re (from STM board manual)" width=50%
```
Comment (dylad) [inline boards/nucleo-l452re/doc.txt:13]:
```suggestion
@image html pinouts/nucleo-l452re.svg "Pinout for the nucleo-l452re (from STM board manual)" width=50%
```

**Review by dylad — CHANGES_REQUESTED** [review]
Spotted a small typo.

File: boards/nucleo-l452re/doc.txt (hunk 1 above)
Comment (krzysztof-cabaj) [reply boards/nucleo-l452re/doc.txt]: Fixed! Squashed!

Thank for spotting this!

**Review by dylad — APPROVED** [review]
Renderer doc looks good.
Thanks for this improvement !

**Pull request merged**